src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Web3
+description: What is web3 and why is it important?
+lang: en
+sidebar: true
+---
+
+# Introduction to web3 {#introduction}
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, these centralized entities have a stronghold on large swathes of the internet, undemocratically deciding what should and should not be allowed.
+
+Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users.
```

Comment:
```suggestion
Web3 is a vision to the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users.
```
Thinking out loud - should we define web3 as a concrete thing? Or is it more of a abstract vision that guides what we're all collectively building towards?

Reply:
Definitely more abstract IMO.